components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -203,13 +203,14 @@ std::string LoadOnFileTaskRunner(
 }
 
 std::vector<ads::AdInfo> GetAdsForCategoryOnFileTaskRunner(
+    const std::string region,
```

Comment:
how granular are the regions? we want to avoid having users uniquely identifiable by region + category